code/game/objects/structures/lattice.dm

```diff
@@ -60,6 +60,11 @@
 	smooth = SMOOTH_TRUE
 	canSmoothWith = null
 
+/obj/structure/lattice/catwalk/New()
+	..()
+	stored.amount++
```

Comment:
The rods that lattices have just needs to be incremented by one. That said, if there's a better way to do it, someone should chime in.